import org.apache.hadoop.fs.PathIOException;
public class FunctionUse19031 {
public void funcUse() {
PathIOException pathioexception = new PathIOException();
pathioexception.getMessage();
}
}